vm-setup/roles/v1aX_integration_test/tasks/healthcheck.yml

```diff
@@ -0,0 +1,185 @@
+# Worker CAPI healthcheck and remediation controller test
+
+- name: Deploy CAPI healthcheck for worker
+  shell: kubectl apply -f "{{ CRS_PATH }}/machine-healthcheck-worker.yaml" -n "{{ NAMESPACE }}"
+
+- name: Verify that CAPI healthcheck is created
+  kubernetes.core.k8s_info:
+    api_version: cluster.x-k8s.io/v1alpha3
+    kind: MachineHealthCheck
+    namespace: "{{ NAMESPACE }}"
+  retries: 20
+  delay: 20
+  register: capi_healthcheck
+  until: (capi_healthcheck is succeeded) and
+         (capi_healthcheck.resources | length == 1)
+
+- name: Fetch the target cluster kubeconfig
+  k8s_info:
+    kind: secrets
+    name: "{{ CLUSTER_NAME }}-kubeconfig"
+    namespace: "{{ NAMESPACE }}"
+  register: metal3_kubeconfig
+
+- name: Decode and save cluster kubeconfig
+  copy:
+    content: "{{ metal3_kubeconfig.resources[0].data.value | b64decode }}"
+    dest: "/tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml"
+
+- name: Install Calico to workflow cluster
+  shell: kubectl --kubeconfig /tmp/kubeconfig-{{ CLUSTER_NAME }}.yaml apply -f https://docs.projectcalico.org/v3.20/manifests/calico.yaml
+
+- name: Pause for 2 minutes. Wait for Nodes to get ready state
+  pause:
+    minutes: 1
```

Comment:
The task name says pause for 2 minutes, but the task only pauses for 1 minute.

Would it be better instead to explicitly wait for all the nodes to become Ready, instead of waiting for an arbitrary time?